tests/main/interfaces-network-status/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Ensure that the network-status interface works.
+
+details: |
+    The network-status interface allows a snap to monitor networking.
+
+    A snap which defines the network-status plug must be shown in the interfaces list.
+    The plug must not be autoconnected on install and, as usual, must be able to be
+    reconnected.
+
+    The snap is also declaring a plug on this interface must be able to ask for its status.
+
+systems: [-ubuntu-core-16-*]
+
+prepare: |
+    . "$TESTSLIB/dbus.sh"
+    . "$TESTSLIB/dirs.sh"
+
+    echo "Given a snap declaring a plug on the network-status interface is installed"
+    snap install test-snapd-network-status-provider
+    snap connect test-snapd-network-status-provider:mount-observe
```

Comment:
Can you please add a comment explaining why the mount-observe interface is relevant for this test? Is it an implementation detail or is this something that this test wants to measure?

Reply:
The snap is trying to access /proc/22413/mounts when we call it from dbus-send, should I add a comment in the code explaining that?

Reply:
Anyway, I have simplified the test and it is not doing anymore the dbus call, just making interactu provider and consumer is enought to validate the interface.